doc/man/lttng-add-context.1.txt

```diff
@@ -46,13 +46,17 @@ can only be used in the Linux kernel tracing domain, while per-thread
 counters can only be used in the user space tracing domain.
 
 It is also possible to enable PMU counters by raw ID using the
-`perf:cpu:raw:r<N>:<name>` or `perf:thread:raw:r<N>:<name>` format for
-the kernel and user-space respectively. `<N>` is a hexadecimal event
-descriptor which is the same format as used by perf-record(1): a
-concatenation of the `Umask value` and `Event number` provided by the
-processor's manufacturer. The possible values for this field are
-processor-specific. The `<name>` field is used to associate a clearer
-name to the counter.
+`perf:cpu:raw:rN:NAME` (Linux kernel tracing domain) or
+`perf:thread:raw:rN:NAME` (user space tracing domain), with:
+
+`N`::
+    A hexadecimal event descriptor which is the same format as used
```

Comment:
Well, I did not disagree with your suggestion to use _An_. It appears that, with some accents, when the first syllable of the word starting with _h_ is not stressed (which is the case with _hexadecimal_, the stress accent being on _dec_), it is correct to use _an_ before the word.

According to [this](http://english.stackexchange.com/questions/629/when-should-i-use-a-versus-an-in-front-of-a-word-beginning-with-the-letter-h), Queen Elizabeth II would say _an hexadecimal event descriptor_, while President Obama would say _a hexadecimal event descriptor_, although I doubt that one or the other would really say that, ever. Specifically, President Obama is known to be only interested in static PMU counters in his sessions of trace investigation.